types/src/transaction/mod.rs

```diff
@@ -1836,6 +1868,16 @@ impl Transaction {
             Transaction::ValidatorTransaction(_) => String::from("validator_transaction"),
         }
     }
+
+    pub fn type_name(&self) -> &'static str {
```

Comment:
where do you use this method?

Reply:
in general, whenever I need to debug something, it is much easier to print the types of transaction, than everything. so nowhere committed, but don't want to be needing to be writing it over and over - it seems like a good utility to have committed.